content/en/monitors/monitor_types/outlier.md

```diff
@@ -16,133 +16,89 @@ further_reading:
+In the following image, a group of hosts is flushing their buffers together, while one host is flushing its buffer slightly later. DBSCAN picks this up as an outlier whereas MAD does not. This is a case where you might prefer to use MAD since the synchronization of the group is just an artifact of the hosts being restarted at the same time. On the other hand, if instead of flushed buffers, the metrics represented a scheduled job that should be synchronized across hosts, DBSCAN would be the correct choice.
 
 {{< img src="monitors/monitor_types/outliers/outliers-flushing.png" alt="outliers flushing" responsive="true" style="width:80%;">}}
 
-#### Scaled vs. Regular Algorithms
-
-DBSCAN and MAD have scaled versions, called ScaledDBSCAN and ScaledMAD, respectively. In most situations, the scaled algorithms behaves the same as their regular counterparts. However, if DBSCAN/MAD algorithms are identifying outliers within a closely clustered group of metrics, and you would like the outlier detection algorithm to scale with the overall magnitude of the metrics, try the scaled algorithms.
-
-Here is a comparison of DBSCAN and ScaledDBSCAN with tolerances of 3 on field data size in a group of Elasticsearch nodes:
-
-{{< img src="monitors/monitor_types/outliers/outliers-scaled-dbscan-es.png" alt="outliers scaled dbscan es" responsive="true" style="width:80%;">}}
-
-Here is an example of MAD and ScaledMAD algorithms for comparing the usable memory in Cassandra hosts. Both have tolerances of 3 and pct of 20:
-
-{{< img src="monitors/monitor_types/outliers/outliers-scaled-mad-cassandra.png" alt="outliers scaled mad Cassandra" responsive="true" style="width:80%;">}}
+### Notifications
 
-### Setting up alerts
+For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][5] page.
 
-When setting up an outlier alert, an important parameter is the size of the time window. If the window size is too large, by the time an outlier is detected, the bad behavior might have been going on for longer than one would like. If the window size is too short, the alerts are not as resilient to unimportant, one-off spikes.
+## Troubleshooting
 
-Both algorithms are set up to identify outliers that differ from the majority of metrics that are behaving similarly. If your hosts exhibit "banding" behavior as shown below (perhaps because each band represents a different shard), we recommend tagging each band with an identifier, and setting up outlier detection alerts on each band separately.
+The outlier algorithms are set up to identify outliers that differ from the majority of metrics that are behaving similarly. If your hosts exhibit "banding" behavior as shown below (maybe each band represents a different shard), we recommend tagging each band with an identifier, and setting up outlier detection alerts on each band separately.
```

Comment:
```suggestion
The outlier algorithms are set up to identify outliers that differ from the majority of metrics that are behaving similarly. If your tag value exhibit "banding" behavior as shown below (maybe each band represents a different shard), Datadog recommends tagging each band with an identifier, and setting up outlier detection to alert on each band separately.
```

Reply:
Would need to be verified by monitors

Reply:
@unclebconnor - Do you have any feedback here or should we ask data science?

Reply:
Hmmm thought I commented on this but must not have clicked save:

`set up to identify outliers that differ from the majority of metrics that are behaving similarly`

could be 
`set up to identify groups that are behaving differently than their peers`